test/powershell/Modules/Microsoft.PowerShell.Management/Get-ComputerInfo.Tests.ps1

```diff
@@ -341,7 +341,9 @@ public static extern int LCIDToLocaleName(uint localeID, System.Text.StringBuild
         $returnValue = $null
         try
         {
-            $returnValue = Get-CimClassPropVal Win32_DeviceGuard $propertyName -namespace 'root\Microsoft\Windows\DeviceGuard' -ErrorAction Stop
+            #Since we want to catch any exceptions here, we won't use the above Get-Cim* helper functions
```

Comment:
I put that comment in. By "above" I meant the two helper functions above. I wanted to express why the function immediately after defining those does not use them. I'll amend or remove the comment if you prefer.